source/topics/cli/how-to-install-cli.md

```diff
@@ -1,10 +1,26 @@
-The Aptible CLI is packaged as a Ruby gem. To install it, you'll first need to make sure you have Ruby installed locally (version 1.9.2 or greater). To find out how to install Ruby on your system, refer to [the installation guide](https://www.ruby-lang.org/en/documentation/installation/).
+## Aptible Toolbelt
+The preferred method of installing the Aptible CLI is via [Aptible Toolbelt](/toolbelt/). The Toolbelt pins dependencies and will usually run much faster than using the gem.
```

Comment:
Might also be worth noting that it's usually more reliable (because it's unaffected by e.g. a broken Ruby install)